pkg/libvirt/domain.go

```diff
@@ -159,6 +159,10 @@ func domainXML(d *Driver, machineType string) (string, error) {
 }
 
 func virtiofsSupported(conn *libvirt.Connect) error {
+	if conn == nil {
+		return drivers.ErrNotSupported
```

Comment:
`return ErrInvalidLibvirtConnection`, or `ErrInvalidArgs` would be more accurate, but I don't think we have such error constants.